### ISSUE TITLE
Training configuration about teacher WRN28-4.

### ISSUE THREAD

**Issue by Abyssaledge** [clovaai/overhaul-distillation#11]
Hi,
    There is a narrow gap between my results and yours on teacher baseline experiment(CIFAR100 and ImageNet). Could you please show me the hyperparameters of  training teacher WRN28-4? Did you use warmup?  dropout?  How about the other params?

**Comment by bhheo** [comment]
Hi
In my codes, setting (d) without distillation loss is same as WRN28-4 teacher training.
Therefore, basic settings is

```
epoch = 200
lr = 0.1
momentum = 0.9
weight_decay = 5e-4
batch_size = 128
```

If you are using same setting, then

- Learning rate schedule
https://github.com/clovaai/overhaul-distillation/blob/f5b9929f5b29806540ecaf5b83435ad594071d17/CIFAR-100/train_with_distillation.py#L136-L144

- Reflection padding 
https://github.com/clovaai/overhaul-distillation/blob/f5b9929f5b29806540ecaf5b83435ad594071d17/CIFAR-100/train_with_distillation.py#L31

- Nesterov gradients
https://github.com/clovaai/overhaul-distillation/blob/f5b9929f5b29806540ecaf5b83435ad594071d17/CIFAR-100/train_with_distillation.py#L137-L138

can be a reason of performance gap.

I didn't use dropout and warmup.
And Mixup data augmentation is only used for teacher training of PyramidNet-200 (240) in setting (e), (f).

**Comment by Abyssaledge** [comment]
Thanks!!! 
Another question,  why do you initiate a couple of new optimizers to adjust learning rate rather than use `scheduler` in Pytorch?  It seems like optimizing information calculated in the past will be abandoned. Is this reasonable?

**Comment by bhheo** [comment]
There is no special reason.
I was not familiar with PyTroch `lr_scheduler`, so I just use hard-coding for lr schedule. I'm sorry about that.
As you said, momentum will be abandoned when learning rate changes.
But, it is just two times for all training and momentum decay is fast on my setting (0.9).
So, effect on performance might be negligible.
